src/docs/product/integrations/notification-incidents/slack/index.mdx

```diff
@@ -49,7 +49,39 @@ In the next section, we'll walk you through configuring your notification settin
 
 ## Configure
 
-Use Slack for [alerts](#alert-rules) regarding issues, environments, deployment, etc.
+Use Slack for notifications and [alerts](#alert-rules) regarding issues, environments, deployment, etc.
+
+### Personal Notifications
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+Let us know if you have feedback: [ecosystem-feedback@sentry.io](mailto:ecosystem-feedback@sentry.io).
+
+</Note>
+
+You can receive personal workflow, deploy, and issue alert notifications from our Slack app. You can manage your [personal notification settings](/product/alerts/notifications/notification-settings/) by navigating to *User Settings > Notifications*.
```

Comment:
Wonder if we should use "Slack integration" rather than "Slack app" to avoid confusion.

```suggestion
You can receive personal workflow, deploy, and issue alert notifications from our Slack integration. Manage your [personal notification settings](/product/alerts/notifications/notification-settings/) by navigating to **User Settings > Notifications**.
```